presto-main/src/main/java/com/facebook/presto/cost/PlanNodeStatsEstimate.java

```diff
@@ -74,6 +77,21 @@ public double getOutputRowCount()
         return outputRowCount;
     }
 
+    @JsonProperty
+    public double getTotalSize()
+    {
+        return totalSize;
+    }
+
+    /**
+     * Only use when getting all columns and meanwhile do not want to
+     * do per-column estimation.
+     */
+    public double getOutputSizeInBytes()
```

Comment:
What if `totalSize` is not set? Should we try to compute it based on the column statistics?

Reply:
It is tricky to compute based on current column statistics because suppose now there are three columns A,B,C and only column A has known statistics, we are only able to estimate the size of column A, because statsNormalizer has removed B and C from the PlanNodeStatsEstimate where nothing can be estimated.

Reply:
I see. Now I undestand the problem.